bitcoin/BRWalletManager.h

```diff
@@ -369,6 +385,30 @@ BRWalletManagerExtractFileServiceTypes (BRFileService fileService,
                                         const char **blocks,
                                         const char **peers);
 
+//
+// Mark: Wallet Sweeper
+//
+
+typedef enum {
+    WALLET_SWEEPER_SUCCESS
+} BRWalletSweeperStatus;
+
+extern BRWalletSweeper // NULL on error
+BRWalletSweeperNew (BRKey *key, uint8_t isSegwit);
+
+extern void
+BRWalletSweeperFree (BRWalletSweeper sweeper);
+
+extern uint64_t
+BRWalletSweeperGetBalance (BRWalletSweeper sweeper);
+
+extern BRWalletSweeperStatus
+BRWalletSweeperHandleTransactionOutput (BRWalletSweeper sweeper,
+                                        UInt256 hash,
+                                        uint32_t index,
+                                        OwnershipKept uint8_t *script,  size_t scriptLen,
+                                        uint64_t satoshis);
+
```

Comment:
Ah, so this updates `sweeper` with the UTXO?

Reply:
Correct. Modelled loosely on the migrator.